src/schemas/ev42/ev42_rw.cpp

```diff
@@ -117,26 +112,65 @@ void HDFWriterModule::parse_config(std::string const &ConfigurationStream,
     Logger->trace("buffer_packet_max: {}", buffer_packet_max);
   } catch (...) { /* it's ok if not found */
   }
+  try {
+    RecordAdcPulseDebugData =
+        ConfigurationStreamJson["adc_pulse_debug"].get<bool>();
+    Logger->trace("adc_pulse_debug: {}", RecordAdcPulseDebugData);
+  } catch (...) { /* it's ok if not found */
+  }
+}
+
+void HDFWriterModule::createAdcGroupAndDatasets(hdf5::node::Group &HDFGroup) {
+  auto AdcGroup = HDFGroup.create_group(AdcGroupName);
+
+  NeXusDataset::Amplitude(        // NOLINT(bugprone-unused-raii)
+      AdcGroup,                   // NOLINT(bugprone-unused-raii)
+      NeXusDataset::Mode::Create, // NOLINT(bugprone-unused-raii)
+      chunk_bytes);               // NOLINT(bugprone-unused-raii)
```

Comment:
Chunk size is in this case the number of elements, not number of bytes. Due to the current state of the ev42 writer module, I suggest that we leave the implementation as is but write a comment (and maybe a JIRA ticket) to clarify this fact and that we intend to fix it in the future.

Reply:
For now I've converted from `chunk_bytes` to a number of elements so that it is clear what is going on in the code and the `chunk_bytes` option in the JSON will do what people expect.
When we refactor the rest of the module we could make the option `chunk_size` instead and avoid the problem.